docs/content/integrations/airlift/reference.mdx

```diff
@@ -90,3 +90,36 @@ In order to make spin-up more efficient, `dagster-airlift` caches the state of t
 ### Automating changes to code locations
 
 If changes to your Airflow instance are controlled via a ci/cd process, you can add a step to automatically induce a redeploy of the relevant code location. See the docs [here](https://docs.dagster.io/concepts/webserver/graphql-client#reloading-all-repositories-in-a-repository-location) on using the graphql client to do this.
+
+### Peering to multiple Airflow instances
+
+Airlift supports peering to multiple Airflow instances, as you can invoke `build_defs_from_airflow_instance` multiple times and combine them with `Definitions.merge`:
+
+```python
```

Comment:
Long run, I think we'll want examples in `docs_snippets` so that we can get linting, but it's a non-blocker.

Reply:
Totally agreed. I have a plan for that, but want to take things one step at a time (since proper test coverage of those kinds of snippets is tricky).